src/index.html

```diff
@@ -1,17 +1,204 @@
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            id="birth" 
+            name="birth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            id="agree" 
+            name="agree"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend> 
+          Registration
+        </legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            id="password" 
+            name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="cat" 
+              value="y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="cat" 
+              value="n"
+            >
+            No
+          </label>
+        </div>
+      
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color" 
+            id="favoriteColor"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="timeToBed" 
+            id="timeToBed"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="brand" 
+            id="brand" 
+            multiple
+          >
+            <option value="BMW">
+              BMW
+            </option>
+            <option value="Audi">
+              Audi
+            </option>
+            <option value="Lada">
+              Lada
+            </option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rateWork" 
+            id="rateWork"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>
+          Additional info:
+        </legend>
+        <label class="form-field">
+          Comments:
+          <textarea 
+            name="comments" 
+            id="comments" 
+            cols="20" 
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="comments" 
            id="comments" 
            cols="20" 
            rows="2"
          ></textarea>
```
![image](https://user-images.githubusercontent.com/96409563/215702532-cf8b77be-73d6-4ee7-bca2-02b7163fd2e7.png)
![image](https://user-images.githubusercontent.com/96409563/215702606-84387986-a646-4ca7-842f-f70ecd9ae489.png)

Reply:
Hello, I fixed the element textarea.